script-templates/bash/README.md

```diff
@@ -90,12 +87,11 @@ curl -s -X POST $BROADCAST_URL -d '{"type":"refresh_lab"}'
 
 A challenge in developing scripts that interact with lab metadata is that it is only available from within a Skytap environment. To help with this, the `bin/run` script runs a "metadata stub" service, simulating the behavior of the Metadata Service and Control Endpoint locally and returning stubbed data. If you would like to modify the stubbed data returned when running your script locally, simply modify the files in `lib/script_support/stub_data`.
 
-**Important**: To ensure that your scripts can be run both locally and in Skytap, please take note of the following:
+**Important**: To ensure that your scripts can be run both locally and in your Course Manager labs, please take note of the following:
 
-* Scripts should always access the Metadata Service via the URL **http://skytap-metadata/skytap**. This is a special URL that only works in Course Manager scripts. This should work in scripts run with `bin/run`, as well as from the Script Host in Course Manager-deployed labs (beginning with Script Host v10).
+* Scripts should always access the Metadata Service via the URL **http://skytap-metadata/skytap**. This is a special URL that only works in Course Manager scripts. This should work in scripts run locally with `bin/run`, as well as from the Script Host in Course Manager-deployed labs (beginning with Script Host v10).
 * If your code accesses to the Lab Control endpoint, note that the endpoint URL used by the Metadata Stub Service will use http, but in production, the Lab Control endpoint will use https. Your code should always retrieve the Lab Control endpoint from the Metadata Service as shown above, expect that it may use either HTTP or HTTPS, and function accordingly.
```

Comment:
Change "If your code accesses to the Lab Control Endpoint" to "If your code accesses the Lab Control Endpoint"